contracts/src/interfaces/IAerodromeLpHyperdrive.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.22;
+
+import { IHyperdrive } from "./IHyperdrive.sol";
+
+interface IAerodromeLpHyperdrive is IHyperdrive {
+    /// @notice Gets the vault used as this pool's yield source.
+    /// @return The compatible yield source.
+    function vault() external view returns (address);
```

Comment:
We should add the `getReward` function to this interface. We don't have a `vault` function.

We also need to add a `gauge` getter to make sure that the immutable is visible externally.